lang_GENERIC/analyze/Naming_AST.ml

```diff
@@ -136,8 +136,17 @@ module V = Visitor_AST
 type resolved_name = AST_generic.resolved_name
 (*e: type [[Naming_AST.resolved_name]] *)
 
+(*s: type [[Naming_AST.entinfo]] *)
+type entinfo = { 
+  (* variable kind and sid *)
+  entname: resolved_name; 
```

Comment:
maybe just name and type, unless they are in conflicts with other fields in other record definition in this file.

Reply:
I had to call them this because I ran into ambiguities otherwise